Daytime-Operations/Auxiliary-Telescope/AT-Daytime-Checkout/Daytime-Checkout-SAL-Scripts.rst

```diff
@@ -0,0 +1,111 @@
+They can also be run individually if a single component is undergoing maintenance 
+and requires a check of basic functionality. 
+
+None of the scripts require configuration. 
+
+.. Important::
+
+    The daytime checkout scripts involve Telescope, Dome, and Instrument component movement. 
+    Before running, ensure that the area surrounding the telescope is and dome is clear for movement. 
+    You MUST announce that you are running the checks on the #summit-announce slack channel before execution. 
+
+The scripts can all be found under the Standard scripts panel. 
+The order of script execution is nominally:
+
+* auxtel/enable_latiss.py - Can be skipped if LATISS enabled 
+* auxtel/daytime_checkout/latiss_checkout.py
+* auxtel/enable_atcs - Can be skipped if ATCS already enabled
+* auxtel/daytime_checkout/atpneumatics_checkout.py
+* auxtel/daytime_checkout/telescope_and_dome_checkout.py
+* auxtel/daytime_checkout/slew_and_take_image_checkout.py
+* auxtel/prepare_for/flats - Can be substituted for other prepare_for scripts to leave AuxTel in different state
+
+*auxtel/enable_latiss.py*
+===============
+
+Bring LATISS to an enabled state to prepare for checkout. 
+
+*auxtel/daytime_checkout/latiss_checkout.py*
+=================
+
+This script will take a bias and engineering frame with LATISS and ensure that they are ingested by the OODS.
+
+Check that the bias pops up in the monitor (https://roundtable.lsst.codes/rubintv/summit/auxtel/monitor_current)
+If it fails, let Patrick or Merlin know.
+
+You can check that it ingested at USDF with this link: ******* <b>Add when available<b>
+
+*auxtel/daytime_checkout/enable_atcs.py*
+=============
+
+Bring the ATCS into an enabled state to prepare for checkout. 
+
+*Standrd script - auxtel/daytime_checkout/atpneumatics_checkout.py*
+=====================
+
+This script will first turn on the valves and check that the line pressure is sufficient for operations. 
+Then, it will turn on/off the ATAOS corrections before lowering the mirror back down onto its hard points.
+Finally the mirror cover and vents are opened and closed. 
+No telescope movement is performed.
+
+*auxtel/daytime_checkout/telescope_and_dome_checkout.py*
+=============================
+
+This script starts by performing a slew of the telescope without dome movement. 
+Then sidereal tracking for the telescope is enabled and the system is left to track for a few minutes. 
+Tracking is disabled and then the dome is commanded to move to a new position, without telescope movement. 
+Finally, the telescope and dome are returned to a park position and left enabled.
+
+*auxtel/daytime_checkout/slew_and_take_image_checkout.py*
+=============================
+
+This script will slew the telescope and dome to two different targets, 
+track each target for 3 minutes and take a single verification image. 
+The mirror covers remain closed during the duration of the script. 
+At the end of the script the Telescope and Dome are parked. 
```

Comment:
Why telescope and dome in capitals here? It's not a big deal but keeps drawing my attention :)